exp/lib/api_utils.py

```diff
@@ -45,6 +45,14 @@ def patch(path, payload=None, params=None):
   response.raise_for_status()
   return response.json()
 
+def put(path, payload=None, params=None):
```

Comment:
Rather than copy and paste, maybe make a common request function that handles the commonality between get/post/put/etc? You could also dynamically call the correct method on requests based on the name of the current function. See: https://github.com/stripe/stripe-python/blob/master/stripe/http_client.py#L98